homeassistant/components/homekit/type_thermostats.py

```diff
@@ -167,7 +168,9 @@
 
 def _hk_hvac_mode_from_state(state: State) -> int | None:
     """Return the equivalent HomeKit HVAC mode for a given state."""
-    if not (hvac_mode := try_parse_enum(HVACMode, state.state)):
+    if (current_state := state.state) == STATE_UNKNOWN:
+        return None
```

Comment:
Can we also handle the `unavailable` state?

Got this after rebooting with this fix:

```
Logger: homeassistant.components.homekit.type_thermostats
Source: components/homekit/type_thermostats.py:174
Integration: HomeKit Bridge ([documentation](https://www.home-assistant.io/integrations/homekit), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+homekit%22))
First occurred: 11:31:40 (4 occurrences)
Last logged: 11:31:41

climate.siterwell_trv: Received invalid HVAC mode: unavailable
climate.saswell_trv_2: Received invalid HVAC mode: unavailable
```

Reply:
Missed your comment before, but yes we should handle both here